css-nav-1/Overview.bs

```diff
@@ -1348,56 +1381,95 @@ Note: the 'spatial-navigation-contain' property is <a>at-risk</a>.
 </div>
 
 <h3 id=css-property-spatialnavigationaction>
-Defining the spatial navigation behavior : the 'spatial-navigation-action' property</h3>
+Controlling the interaction with scrolling: the 'spatial-navigation-action' property</h3>
 
 <pre class='propdef'>
 Name: spatial-navigation-action
 Value: auto | focus | scroll
 Initial: auto
-Applies to: <a>scroll container</a>
-Inherited: true
+Applies to: <a>scroll containers</a>
+Inherited: no
 </pre>
 
-When the user triggers spatial navigation in a given direction, the 'spatial-navigation-action' decides
-the proper spatial navigation behavior.
+When the focus is inside of a scroll container and the user triggers spatial navigation,
+it is somewhat ambiguous whether they are requesting that the focus be moved in that direction,
+or whether the document should be scrolled in that direction.
+By default, this is automatically determined,
+but this property allows the author decide between focusing or scrolling.
+
+The precise behavior is defined in [[#nav]],
+but a high level description of the effect of each value is provided below.
+
+When spatial navigation is triggered,
+the behavior depends on the value of the 'spatial-navigation-action' on the currently focused element
+if that element is a [=scroll container=],
+or of its nearest [=scroll container=] ancestor if it isn't.
 
 <dl dfn-for=spatial-navigation-action dfn-type=value>
     <dt><dfn>auto</dfn>
-    <dd>If the element is a <a>scroll container</a>,
-        then directional input moves the focus to the candidates in the optimal viewing region.
-        When there isn't any visible candidate, then the container scrolls.
-        
-        Otherwise, the focus moves to the candidates in the viewport.
+    <dd>
+        If there are visible focusable elements within the
+        [=scroll container=] in the direction requested,
+        the closest one becomes focused.
+        Otherwise, the [=scroll container=] is scrolled in the direction requested.
 
     <dt><dfn>focus</dfn>
-    <dd>The directional input moves the focus to all candidates inside a scroll container. The container <a>cannot be scrolled manually</a>.
+    <dd>
+        The focus is moved to the nearest focusable element within the [=scroll container=],
+        regardless of whether it is visible.
+        If there are none,
+        the scroll container is <em>not</em> scrolled,
+        and the search continues up the tree instead.
```

Comment:
"up the tree" seems a little bit ambiguous.
How about "up the ancestry chain" that is already used in the spec?
(In the overview section)

Reply:
good idea.